manifests/config.pp

```diff
@@ -54,6 +55,9 @@
   if $https_ports {
     create_resources('squid::https_port', $https_ports)
   }
+  if $snmp_listen_ip {
+    create_resources('squid::snmp_listen_ip', $snmp_listen_ip)
```

Comment:
`create_resources` takes a hash whilst `snmp_listen_ip` is a string.

Reply:
i changed it, so it should run well.
Could you maybe take a look why it tests still fail? I didn't get the reason.

Reply:
As a class, you couldn't use `create_resources`, but you perhaps instead just declare your class here a bit like..
```puppet
class { 'squid::snmp_listen_ip':
  listen_ip => $snmp_listen_ip,
}
```
I'm sorry I've not had the time yet to properly follow up on your PRs.  Hopefully we'll be able to get some more merged soon.  You might find it easier to reach out on #voxpupuli on IRC or on the vox pupuli slack channel.